WordPress/gutenberg
# Testing: Fix and update block fixture regeneration

Regressions of: #6311, #6351

In master, running `npm run fixtures:regenerate` has the result of:

- Not actually regenerating anything, because the current fixtures aren't first removed since the move of `blocks/library` to `core-blocks`
- Introducing an unintended change to `server-registered.json` because the attributes of Latest Post block have been updated

Both of these have been resolved here.

__Testing instructions:__

Verify that `npm run fixtures:clean` actually removes files.

Verify that `npm run fixtures:regenerate` introduces no changes.
